pkgs/development/tools/misc/premake/5.nix

```diff
@@ -10,49 +10,58 @@
 
 stdenv.mkDerivation rec {
   pname = "premake5";
-  version = "5.0.0-beta4";
+  version = "5.0.0-beta6";
 
   src = fetchFromGitHub {
     owner = "premake";
     repo = "premake-core";
     rev = "v${version}";
-    sha256 = "sha256-sNLCyIHWDW/8jIrMFCZAqtWsh4SRugqtPR4HaoW/Vzk=";
+    sha256 = "sha256-wMy9uqtlWAB+o/D169CLNQDd80zhsWKyNBXzKxhH7n4=";
   };
 
   buildInputs =
-    [ libuuid ]
+    [
+      libuuid
+    ]
     ++ lib.optionals stdenv.hostPlatform.isDarwin [
       Foundation
       readline
     ];
 
-  patches = [ ./no-curl-ca.patch ];
+  # patches = [ ./no-curl-ca.patch ];
   postPatch =
     ''
       substituteInPlace contrib/curl/premake5.lua \
-        --replace "ca = nil" "ca = '${cacert}/etc/ssl/certs/ca-bundle.crt'"
+        --replace-fail "ca = nil" "ca = '${cacert}/etc/ssl/certs/ca-bundle.crt'"
+      substituteInPlace premake5.lua \
```

Comment:
quite fragile. I'd prefer to see a patch for this.

Reply:
I'm not sure this is readily patchable as `${cacert}` isn't static. Might be worth an upstream patch, if this becomes used more widely.

Reply:
Ah. I've seen a hack for these situations where the patch patches in with a template string that is then replaced with --replace-fail. But it's a bit tricky and probably not worth doing right now.